src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery(new ColorSupplier(), new Random(), 100);
```

Comment:
Let's isolate Lottery logic and don't pass `new ColorSupplier(), new Random(), 100` to it.
```suggestion
        Lottery lottery = new Lottery();
```